projects/packages/forms/src/blocks/contact-form/components/jetpack-field-choice/item/edit.js

```diff
@@ -1,13 +1,64 @@
-import { RichText, useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
-import { createBlock } from '@wordpress/blocks';
+import {
+	RichText,
+	useBlockProps,
+	useInnerBlocksProps,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { createBlock, cloneBlock, getDefaultBlockName } from '@wordpress/blocks';
+import { useRefEffect } from '@wordpress/compose';
 import { useDispatch, useSelect } from '@wordpress/data';
+import { useRef } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import clsx from 'clsx';
-import { first } from 'lodash';
 import { supportsParagraphSplitting } from '../../../util/block-support';
 import { useParentAttributes } from '../../../util/use-parent-attributes';
 import { useJetpackFieldStyles } from '../../use-jetpack-field-styles';
 
+function useEnter( props ) {
+	const { replaceBlocks, selectionChange } = useDispatch( blockEditorStore );
+	const { getBlock, getBlockRootClientId, getBlockIndex } = useSelect( blockEditorStore );
+	const propsRef = useRef( props );
+	propsRef.current = props;
+	return useRefEffect( element => {
+		function onKeyDown( event ) {
+			// Keycode for ENTER is `13`.
+			if ( event.defaultPrevented || event.keyCode !== 13 ) {
```

Comment:
`import { ENTER } from '@wordpress/keycodes';` 

Should we use the ENTER constant here?

Reply:
At first I didn't want to add the new dependency, but it's better to use `key` instead as [keyCode is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).